components/elm/src/dyn_subgrid/dynFATESLandUseChangeMod.F90

```diff
@@ -0,0 +1,187 @@
+module dynFATESLandUseChangeMod
+
+#include "shr_assert.h"
+
+  !---------------------------------------------------------------------------
+  ! !DESCRIPTION:
+  ! Handle reading of the land use harmonization (LUH2) dataset
+
+  ! !USES:
+  use shr_kind_mod          , only : r8 => shr_kind_r8
+  use shr_log_mod           , only : errMsg => shr_log_errMsg
+  use decompMod             , only : bounds_type, BOUNDS_LEVEL_PROC
+  use abortutils            , only : endrun
+  use dynFileMod            , only : dyn_file_type
+  use dynVarTimeUninterpMod , only : dyn_var_time_uninterp_type
+  use elm_varcon            , only : grlnd
+  use elm_varctl            , only : iulog
+
+  implicit none
+
+  private
+
+  real(r8), allocatable, public :: landuse_transitions(:,:)
+  real(r8), allocatable, public :: landuse_states(:,:)
+
+  integer, public, parameter    :: num_landuse_transition_vars = 108
+  integer, public, parameter    :: num_landuse_state_vars = 12
+
+  type(dyn_file_type), target   :: dynFatesLandUse_file
+
+  ! Land use name arrays
+  character(len=5), public, parameter  :: landuse_state_varnames(num_landuse_state_vars) = &
+                    [character(len=5)  :: 'primf','primn','secdf','secdn','pastr','range', &
+                                          'urban','c3ann','c4ann','c3per','c4per','c3nfx']
```

Comment:
How about adding the long name for these in the comment?

```
  character(len=5), public, parameter  :: landuse_state_varnames(num_landuse_state_vars) = &
                    [character(len=5)  :: 'primf', & ! Primary forest 
                                          'primn', & ! Primary something???
```

Reply:
That's a good idea.  I could also point developers to the [data readme](https://luh.umd.edu/LUH2/LUH2_v2h_README.pdf) location online.  How do we feel about having urls in the code comments?

Reply:
I believe the URLs are discouraged because they would inevitably get broken in the future.

Reply:
Updated via f09f5e3b8bf54ce5eb13e7597946cfbcf4a7f39e